Product/Production/Common/CONNECTCoreLib/src/test/java/gov/hhs/fha/nhinc/callback/openSAML/SAMLAssertionBuilderFactoryImplTest.java

```diff
@@ -35,9 +35,9 @@
     @Test
     public void getBuilderTest() {
```

Comment:
this PR addresses issues from sonar report. refactoring the test conditions may be out of scope